cloudbuild/google/cloud/cloudbuild_v1/proto/cloudbuild.proto

```diff
@@ -0,0 +1,1304 @@
+  string machine_type = 1;
+
+  // Size of the disk attached to the worker, in GB.
+  // See https://cloud.google.com/compute/docs/disks/
+  // If `0` is specified, Cloud Build will use a standard disk size.
+  // `disk_size` is overridden if you specify a different disk size in
+  // `build_options`. In this case, a VM with a disk size specified in the
+  // `build_options` will be created on demand at build time. For more
+  // information see
+  // https://cloud.google.com/cloud-build/docs/api/reference/rest/v1/projects.builds#buildoptions
+  int64 disk_size_gb = 2;
+
+  // The network definition used to create the worker.
+  // If this section is left empty, the workers will be created in
+  // WorkerPool.project_id on the default network.
+  Network network = 3;
+
+  // The tag applied to the worker, and the same tag used by the firewall rule.
+  // It is used to identify the Cloud Build workers among other VMs.
+  // The default value for tag is `worker`.
+  string tag = 4;
+}
+
+// Network describes the GCP network used to create workers in.
+message Network {
+  // Project id containing the defined network and subnetwork. For a peered VPC,
+  // this will be the same as the project_id in which the workers are created.
+  // For a shared VPC, this will be the project sharing the network with the
+  // project_id project in which workers will be created. For custom workers
+  // with no VPC, this will be the same as project_id.
+  string project_id = 1;
+
+  // Network on which the workers are created.
+  // "default" network is used if empty.
+  string network = 2;
+
+  // Subnetwork on which the workers are created.
+  // "default" subnetwork is used if empty.
+  string subnetwork = 3;
+}
+
+// Request to create a new `WorkerPool`.
+message CreateWorkerPoolRequest {
+  // ID of the parent project.
+  string parent = 1;
+
+  // `WorkerPool` resource to create.
+  WorkerPool worker_pool = 2;
+}
+
+// Request to get a `WorkerPool` with the specified name.
+message GetWorkerPoolRequest {
+  // The field will contain name of the resource requested, for example:
+  // "projects/project-1/workerPools/workerpool-name"
+  string name = 1;
+}
+
+// Request to delete a `WorkerPool`.
+message DeleteWorkerPoolRequest {
+  // The field will contain name of the resource requested, for example:
+  // "projects/project-1/workerPools/workerpool-name"
+  string name = 1;
+}
+
+// Request to update a `WorkerPool`.
+message UpdateWorkerPoolRequest {
+  // The field will contain name of the resource requested, for example:
+  // "projects/project-1/workerPools/workerpool-name"
+  string name = 2;
+
+  // `WorkerPool` resource to update.
+  WorkerPool worker_pool = 3;
+}
+
+// Request to list `WorkerPool`s.
+message ListWorkerPoolsRequest {
+  // ID of the parent project.
+  string parent = 1;
+}
+
+// Response containing existing `WorkerPools`.
+message ListWorkerPoolsResponse {
+  // `WorkerPools` for the project.
+  repeated WorkerPool worker_pools = 1;
+}
```

Comment:
The `cloudbuild_pb2.py` file is missing.

Reply:
Moved `cloudbuild.proto` to `google/cloud/devtools/cloudbuild_v1/proto/cloudbuild.proto` (and removed `google/cloud/cloudbuild_v1`.